src/site/_site/Documentation/index.js

```diff
@@ -22,13 +22,13 @@ export default class Documentation extends React.Component {
         <h3 className="Documentation_heading">Examples</h3>
         <div className="Documentation__section">{children}</div>
         <h3 className="Documentation_heading">Documentation</h3>
-        <div
-          className="Documentation__section"
-          dangerouslySetInnerHTML={HTML_DOCS}
-        />
+        <div className="Documentation__section" dangerouslySetInnerHTML={HTML_DOCS} />
       </div>
     ) : (
-      <div className={className}>{children}</div>
+      <div className={className}>
+        <h1 className={['pf-u-sr-only']}>Bare Component Demo</h1>
```

Comment:
This can be written as `className="pf-u-sr-only"` I am actually surprised that works.  I don't think it is valid syntax.  If someone adds another class to that array my guess is that it will be joined with `,`.

Outside of the scope of this PR, but it would be nice to have the title of the example/component here. That way it is more useful overall.

Reply:
Typically with this syntax, you'd be using a single property name and it refers to an array in you controller, of course using `.join(' ')`, this gives one an easy way to add/remove classes dynamically. Since it's only a single class we need, though, and we'll probably not add more I'm good with updating to use standard className syntax.